### PR TITLE
View comments list

### PR DESCRIPTION
**Pull request by AnyaVeselova**
* creates CommentsList and Comment components
* creates fetchCommentsByArticleId util function
* styles comments section using MUI
* changes styling of votes in Article component
* sets loading element in ArticleList component

### CODE REVIEW

File: src/components/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  CardHeader,
+  CardContent,
+  Typography,
+  Box,
+  IconButton,
+} from "@mui/material";
+import { ThumbUp, ThumbDown } from "@mui/icons-material";
+
+export function Comment({ comment }) {
+  console.log(comment);
```
Comment (nc-modou) [inline src/components/Comment.jsx:13]: Remember to remove console logs once you're done debugging

File: src/components/CommentsList.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      }}
+    >
+      {comments.map((comment) => (
+        <Grid item key={comment.comment_id}>
+          <Link
+            to={`/comments/${comment.comment_id}`}
+            style={{ textDecoration: "none" }}
+          >
+            <Comment comment={comment} />
+          </Link>
```
Comment (nc-modou) [inline src/components/CommentsList.jsx:48]: Comments look great, but there's no need for clicking it to take you to a separate page :)
It's already showing all the relevant data and it doesn't really make sense to have a page for each comment

**Review by nc-modou — COMMENTED** [review]
The comments look great so far Anna!

One thing I did notice is that an article with no comments has a loading spinner that never stops in the comment section - would be ideal to have a good display for an article that has no comments. Something like a little ' no comments yet' message would be great.

File: src/components/CommentsList.jsx (hunk 2 above)
Comment (AnyaVeselova) [reply src/components/CommentsList.jsx:48]: What if I want to create an option to comment on the comment in the future? :)

**Pull request merged**